docs/cli.rst

```diff
@@ -64,6 +64,11 @@ also remove the fragments directory if now empty.
 
    Don't delete news fragments after the build and don't ask for confirmation whether to delete or keep the fragments.
 
+.. option:: --strict
```

Comment:
Do we need this argument ?

I think that having `build_ignore_filenames` is enough.